utils/filters.ts

```diff
@@ -15,7 +15,8 @@ type VisibleHidden = {
 
 //
 
-const meatTags = [ 'S', 'R', 'G', 'L', 'W', 'F', 'MSC', 'Kr', 'We' ]
+const fishTags = [ 'Fi', 'F', 'MSC' ]
+const meatTags = [ ...fishTags, 'S', 'R', 'G', 'L', 'W', 'Kr', 'We' ]
```

Comment:
I don't like adding fish tags to meat tags. Right now you can chose to show fish dishes while hiding meat, this would no longer be possible without any benefit (except for maybe having to flip one switch instead of two in the settings). Additionally the meat filter now essentially works the same as the "only show vegetarian" filter.

Reply:
sounds good